src/sentry/static/sentry/app/components/modals/createTeamModal.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {Client} from '../../api';
+import {createTeam} from '../../actionCreators/teams';
+import {t} from '../../locale';
+import CreateTeamForm from '../createTeam/createTeamForm';
+import SentryTypes from '../../proptypes';
+
+class CreateTeamModal extends React.Component {
+  static propTypes = {
+    closeModal: PropTypes.func,
+    onClose: PropTypes.func,
+    Body: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
+    Header: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    project: SentryTypes.Project,
+  };
+
+  handleSubmit = data => {
+    createTeam(new Client(), data, {orgId: this.props.organization.slug}).then(
```

Comment:
should these action creators just manage their own api client now?

Reply:
I'm not sure right now, I wanted to keep the API the same between the team action creators (they all accept api client as the first arg). If they were to handle their own API client, components lose the ability to easily cancel API requests (how often this happens in practice should probably be measured).